proxy-integration-test.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
little confused how this file is different from the integration test I already implemented? (proxy-test.sh)